gamemodes/zombiesurvival/gamemode/shared.lua

```diff
@@ -787,7 +787,7 @@ end
 
 function GM:GetRagdollEyes(pl)
 	local Ragdoll = pl:GetRagdollEntity()
-	if not Ragdoll then return end
+	if not Ragdoll:IsValid() then return end
```

Comment:
Player.GetRagdollEntity can return nil so it should be
if not Ragdoll or not Ragdoll:IsValid() then return end